pt_BR Last commit for oct/22

M-art-ucci opened the pull request:
Just few more adjustments revised for better reading

M-art-ucci committed:
Update pt_BR.json

AUTOMATIC1111 commented:
i added you to collaborators and added the file as owned by you to codeowners. See if you can can accept this pull request yourself.

M-art-ucci commented:
> i added you to collaborators and added the file as owned by you to codeowners. See if you can can accept this pull request yourself.

Sorry about the workload. I'm new to this. Thanks in advcane you for being patient.

It seems I can. Thanks again!

M-art-ucci commented:
@AUTOMATIC1111 actually I didn't find where to commit the changes to merge them and I closed the PR.

When you have the time, can you tell me if you can see this message?

Sorry about the mess.
